client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -88,6 +91,13 @@ class CalypsoifyIframe extends Component {
 
 			// Check if we're generating a post via Press This
 			this.pressThis();
+			return;
+		}
+		if ( 'classicBlockOpenMediaModal' === action ) {
```

Comment:
I think we can reuse the existing `openMediaModal` action. You'll need to add additional data to the payload (i.e. the block editor id) but we'll avoid to maintain two media modal components. Also, that will probably provide for free the edit functionality you're missing right now.